mlflow/server/js/src/components/BaggedCell.js

```diff
@@ -16,16 +17,17 @@ export default class BaggedCell extends PureComponent {
   static propTypes = {
     keyName: PropTypes.string.isRequired,
     value: PropTypes.string.isRequired,
-    setSortByHandler: PropTypes.func.isRequired,
+    onSortBy: PropTypes.func.isRequired,
     isParam: PropTypes.bool.isRequired,
     isMetric: PropTypes.bool.isRequired,
     onRemoveBagged: PropTypes.func.isRequired,
     sortIcon: PropTypes.node,
   };
 
   render() {
-    const { keyName, value, setSortByHandler, isParam, isMetric, onRemoveBagged,
-      sortIcon } = this.props;
+    const { keyName, value, onSortBy, isParam, onRemoveBagged, sortIcon } = this.props;
+    const keyType = (isParam ? "params" : "metrics");
+    const canonicalKey = ExperimentViewUtil.makeCanonicalKey(keyType, keyName);
```

Comment:
We introduce the notion of a canonical key, like

```
metrics.`foo`
```

This allows a column to directly reference the data that produces it.